bramkouwen/Music-Mixer
# Removing files in Assets folder

## What does the PR do?
* It removes the PDFs from assets folder

## How do you check this PR?
* Pull down the branch
*  See removed files

## Screenshots 
None.
